src/main/resources/db/migration/common/V1_413__DETSCleanupArmResponseFiles_automated_task.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size,
+created_ts, created_by,last_modified_ts,last_modified_by, task_enabled)
+VALUES (nextval('aut_seq'),'DETSCleanupArmResponseFiles','Clean up data from ARM response folders Asynchronously. ','0 27 23 * * ?',true,4000,
```

Comment:
Automated jobs are now using hardcoded values so don't use nextVal. You may also need to add a matching user for the job but not sure if that code has yet been merged.

Reply:
Also mention in the description it is for dets as the admin returns the descriptions of tasks so they can be run

Reply:
Updated